test/test_toolbar.py

```diff
@@ -7,7 +7,6 @@
 
 def load_app(name):
     app = __import__(name).app
-    app.config['TESTING'] = True
```

Comment:
I thought it was best practice to set this when in tests? 

Looks like that's still true for Flask 3.0.x: https://flask.palletsprojects.com/en/3.0.x/testing/

I realize the main side-effect of propagating exceptions also happens when `DEBUG` is `True`, but still seems best to follow best practices as later on additional side effects may be added.

Overall, the presence/absece of this seems like it shouldn't whether the tests pass/fail which is the purpose of this PR, so IMO I'd extract this change to a separate PR.

Reply:
Thanks for the review. I guess I made a mistake here. I don't need to remove this line, but just add the `debug = True` line. Updated.